homeassistant/components/octoprint/__init__.py

```diff
@@ -95,13 +51,6 @@ def ensure_valid_path(value):
                         vol.Optional(CONF_SSL, default=False): cv.boolean,
                         vol.Optional(CONF_PORT, default=80): cv.port,
                         vol.Optional(CONF_PATH, default="/"): ensure_valid_path,
-                        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Comment:
We should keep the config schema intact until we remove the import of the yaml config to the config flow unless we've actually already changed the allowed configuration.yaml.

Reply:
These values won't be doing anything with the config flow import. The flow ignores the selected sensors and the configured name.

Reply:
Good. Then we can restore the schema. We don't want to change this since users should not change anything about their configuration.yaml besides removing it completely.

Reply:
Restored.